src/core/components/deeplinking-link.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+export const DeepLinkingLink = ({ isDeepLinkingEnabled, path, text }) => {
+    return (
+        <a className="nostyle"
+          onClick={isDeepLinkingEnabled ? (e) => e.preventDefault() : null}
```

Comment:
Similar to my comment on the name...
Can we just simplify all these `isDeepLinkingEnabled` to just `enabled` we are already under the deep umbrella